### PR TITLE
service: Update service function tables

### PR DESCRIPTION
**Pull request by Morph1984**
Updates various service functions for these services to 12,x:

btdrv, btm, erpt, vi, bgtc, npns, nim, aoc:u, ns

Also implements bgtc:t ITaskServiceRoot OpenTaskService

### CODE REVIEW

File: src/core/hle/service/btdrv/btdrv.cpp [hunk 1]
```diff
@@ -156,6 +156,25 @@ class BtDrv final : public ServiceFramework<BtDrv> {
             {97, nullptr, "RegisterBleHidEvent"},
             {98, nullptr, "SetBleScanParameter"},
             {99, nullptr, "MoveToSecondaryPiconet"},
+            {100, nullptr, "Unknown100"},
```
Comment (german77) [inline src/core/hle/service/btdrv/btdrv.cpp:159]: They have proper names now

**Comment by german77** [comment]
Missing unknown function 16 in bgtc:t

**Comment by german77** [comment]
Missing unknown function 31 in IApplicationManagerInterface ns.cpp

File: src/core/hle/service/ns/ns.cpp [hunk 2]
```diff
@@ -441,7 +463,11 @@ IApplicationVersionInterface::IApplicationVersionInterface(Core::System& system_
         {800, nullptr, "RequestVersionList"},
         {801, nullptr, "ListVersionList"},
         {802, nullptr, "RequestVersionListData"},
+        {900, nullptr, "Unknown900"},
+        {901, nullptr, "Unknown901"},
+        {902, nullptr, "Unknown902"},
```
Comment (german77) [inline src/core/hle/service/ns/ns.cpp:468]: This have proper names now

**Comment by german77** [comment]
Missing in ns.cpp
7988 | [6.0.0+] GetDynamicRightsInterface
7989 | [5.1.0+] GetReadOnlyApplicationControlDataInterface
7991 | [5.0.0+] GetReadOnlyApplicationRecordInterface

File: src/core/hle/service/ns/ns.cpp [hunk 3]
```diff
@@ -587,6 +613,10 @@ class NS_DEV final : public ServiceFramework<NS_DEV> {
             {13, nullptr, "CreateApplicationResourceForDevelop"},
             {14, nullptr, "IsPreomiaForDevelop"},
             {15, nullptr, "GetApplicationProgramIdFromHost"},
+            {16, nullptr, "Unknown16"},
```
Comment (german77) [inline src/core/hle/service/ns/ns.cpp:616]: This now have proper names. Function 9 and 12 got different names

File: src/core/hle/service/aoc/aoc_u.cpp [hunk 4]
```diff
@@ -118,8 +118,10 @@ AOC_U::AOC_U(Core::System& system_)
         {7, &AOC_U::PrepareAddOnContent, "PrepareAddOnContent"},
         {8, &AOC_U::GetAddOnContentListChangedEvent, "GetAddOnContentListChangedEvent"},
         {9, nullptr, "GetAddOnContentLostErrorCode"},
+        {10, nullptr, "GetAddOnContentListChangedEventWithProcessId"},
         {100, &AOC_U::CreateEcPurchasedEventManager, "CreateEcPurchasedEventManager"},
         {101, &AOC_U::CreatePermanentEcPurchasedEventManager, "CreatePermanentEcPurchasedEventManager"},
+        {110, nullptr, "Unknown110"},
```
Comment (german77) [inline src/core/hle/service/aoc/aoc_u.cpp:124]: This is now called CreateContentsServiceManager

**Review by german77 — APPROVED** [review]

**Review by bunnei — APPROVED** [review]

**Pull request merged**